packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -151,12 +157,12 @@ export interface DropdownProps extends UIComponentProps<DropdownProps, DropdownS
 }
 
 export interface DropdownState {
-  isOpen?: boolean
-  value: ShorthandValue | ShorthandValue[]
-  backspaceDelete: boolean
```

Comment:
We don't need backspaceDelete anymore? Are we going to still have the functionality to remove selected items on backspace?

Reply:
yes :D

Reply:
I think it's because I changed the handler from keyUp to keyDown. it worked, so I just removed that code as it was just a hack to begin with.